### PR TITLE
docs: project update

### PR DESCRIPTION
**Pull request by Rubyt0x**
Added the following projects:
- CoChilli
- Connext
- CryptoSage
- Envio
- Griffy
- Kassiopea
- Orbiter Finance
- Push Protocol
- Pyth
- Seda Protocol
- Sentio
- Subsquid
- Sway Farm
- Sway Playground
- The Graph

Edited the following project logos:
- Bako Identity
- Poolshark

Removed the following projects:
- Elix
- SmartMoney
- Fuel Punks
- Fuel Monkeys
- Unic
- BaldBoyz
- Bulldogs
- FuelID
- FuelNomen
- FuFi
- Quid
- RetroFuel
- Ruscet
- Yama

### CODE REVIEW

File: packages/app-portal/src/systems/Ecosystem/data/projects.json [hunk 1]
```diff
@@ -89,14 +89,36 @@
...
-    "name": "Elix Finance",
-    "url": "https://elix.finance/",
-    "tags": ["DeFi"],
-    "description": "Elix Finance is a hybrid automated market maker featuring automated concentrated liquidity.",
-    "github": "https://github.com/elixfinance",
-    "twitter": "https://twitter.com/elixfinance",
-    "discord": "https://discord.gg/avrdk2tAY6",
-    "image": "elix"
+    "name": "Connext",
+    "url": "https://www.connext.network/",
```
Comment (helciofranco) [inline packages/app-portal/src/systems/Ecosystem/data/projects.json:93]: (nit) just to use the same displaying pattern

```suggestion
    "url": "https://connext.network/",
```

if you don't mind, I realized there are only other 9 using `www.` 

Could you please remove them?

File: packages/app-portal/src/systems/Ecosystem/data/projects.json [hunk 2]
```diff
@@ -144,8 +166,8 @@
   },
   {
     "isLive": true,
-    "name": "Fuel-Bridge",
-    "url": "https://alpha.fuel.network/bridge",
+    "name": "Fuel Bridge",
+    "url": "https://app.fuel.network/portal/bridge",
```
Comment (helciofranco) [inline packages/app-portal/src/systems/Ecosystem/data/projects.json:170]: we've removed this `/portal` 

```suggestion
    "url": "https://app.fuel.network/bridge",
```

File: packages/app-portal/src/systems/Ecosystem/data/projects.json [hunk 3]
```diff
@@ -342,14 +397,25 @@
...
+    "description": "SEDA is a permissionless modular data layer building the new standard for data transport and querying. Natively serving 230+ blockchains SEDA allows protocols to transport, query and configure any data available from any source.",
+    "github": "",
+    "twitter": "https://twitter.com/sedaprotocol",
+    "discord": "https://discord.com/invite/seda",
+    "image": "seda"
+  },
+  {
+    "isLive": false,
+    "name": "Sentio",
+    "url": "https://www.sentio.xyz/",
```
Comment (helciofranco) [inline packages/app-portal/src/systems/Ecosystem/data/projects.json:412]: (nit)

```suggestion
    "url": "https://sentio.xyz/",
```

**Review by helciofranco — APPROVED** [review]
LGTM

File: packages/app-portal/src/systems/Ecosystem/data/projects.json (hunk 1 above)
Comment (Rubyt0x) [reply packages/app-portal/src/systems/Ecosystem/data/projects.json]: Yep will do

**Review by helciofranco — APPROVED** [review]

**Review by LuizAsFight — APPROVED** [review]

**Review by helciofranco — APPROVED** [review]

**Pull request merged**